src/Attributes.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-view for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-view/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-view/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\View;
+
+use ArrayObject;
+use Laminas\Escaper\Escaper;
+use Traversable;
+
+/**
+ * Class for storing and processing HTML tag attributes.
+ */
+class Attributes extends ArrayObject
+{
+    /**
+     * HTML escaper
+     *
+     * @var Escaper
+     */
+    protected $htmlEscaper;
+
+    /**
+     * HTML attribute escaper
+     *
+     * @var Escaper
+     */
+    protected $htmlAttrEscaper;
```

Comment:
No need to use abbreviations here, use `$htmlAttributeEscaper` instead of `$htmlAttrEscaper`.